assets/js/googlesitekit/modules/datastore/settings.js

```diff
@@ -25,7 +25,12 @@ import invariant from 'invariant';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-const { createRegistrySelector } = Data;
+// import { dispatch } from '@wordpress/data';
+const { createRegistrySelector, createRegistryControl } = Data;
+// import { STORE_NAME } from './constants';
+import { STORE_NAME as CORE_MODULES } from './constants';
+
+const SETTINGS_SUBMIT_CHANGES = 'SETTINGS_SUBMIT_CHANGES';
```

Comment:
Let's update the action type here to be a little more specific to this store:

```suggestion
const SUBMIT_MODULE_CHANGES = 'SUBMIT_MODULE_CHANGES';
```

Reply:
This comment is yet to be addressed.

Reply:
Updated.